projects/plugins/jetpack/changelog/add-pocket-casts-dev-block

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: other
+
+Add pocket casts dev block (for internal use only).
```

Comment:
Maybe we could add a "category" to specify where what is the change related to, e.g.:

```suggestion
Blocks: Add Pocket Casts dev block (for internal use only)
```

Reply:
Thanks!